Gemfile

```diff
@@ -23,4 +6,13 @@ else
   gem 'puppet', :require => false
 end
 
-# vim:ft=ruby
+gem 'metadata-json-lint'
+gem 'puppetlabs_spec_helper', '>= 0.1.0'
+gem 'puppet-lint', '>= 1.0.0'
+gem 'facter', '>= 1.7.0'
+gem 'rspec-puppet'
```

Comment:
suggest adding the community puppet-lint plugins

https://github.com/ghoneycutt/puppet-module-nfs/blob/master/Gemfile#L14